src/JWT/Compatibility/CustomAttributeExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿#if NET35
+namespace System.Reflection
+{
+    internal static class CustomAttributeExtensions
+    {
+        public static T GetCustomAttribute<T>(this MemberInfo element) where T : Attribute
+        {
+            return (T)Attribute.GetCustomAttribute(element, typeof(T));
+        }
+    }
+}
+#endif
```

Comment:
Would be great if you could remove the extra line breaks from all new files. Again, OCD :)